examples/intel/file.py

```diff
@@ -11,7 +11,7 @@
 
 
 def main():
-    print(f"Checking file...")
+    print(f"Checking hash...")
```

Comment:
and then we could have these calls go through a logger

Reply:
Yes, for sure we can, in spite of that I would prefer to keep it as simple as prints and that the user may remove them easily or if they need add a logger or whatever that fit their system